conf/locale/es_419/LC_MESSAGES/djangojs.po

```diff
@@ -2903,9 +2907,11 @@ msgstr "¿Deseas abandonar este equipo?"
 
 #: lms/djangoapps/teams/static/teams/js/views/team_profile.js
 msgid ""
-"If you leave, you can no longer post in this team's discussions.Your place "
+"If you leave, you can no longer post in this team's discussions. Your place "
```

Comment:
Here an addition of space after period, does it exists in source string ? Please verify is this working for you.

Reply:
Fixed and verified1